### ISSUE TITLE
Registering user hangs forever with "Please wait" message

### ISSUE THREAD

**Issue by valmar** [RocketChat/Rocket.Chat#12964]
### Description:

If a user tries to register, at the moment when the use clicks the "Register new account" button, the message "Please wait" appears, and the whole system hangs there, without ever moving forward.

If I log in as administrator, I see that the user has actually been created.

### Steps to reproduce:

1. Try to register as a new user
2. Click on "Register a new account"

### Expected behavior:

The registration ends successfully.

### Actual behavior:

The message "Please wait" appears and the system hangs

### Server Setup Information:

- Version of Rocket.Chat Server: 0.72.3
- Operating System: CentOS 7.5.1804
- Deployment Method: Manual Install
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: v8.9.3
- MongoDB Version: 2.6.12

### Additional context

Could be related or not. If the newly created user goes back to the login screen and tries to login, the system hangs after he selects the username, again with the message "Please Wait". Furthermore, if I as administrator try and set the username manually, pressing the "Save" button in the user detail screen has no effect...

### Relevant logs:

No errors in RocketChat or Nginix logs

**Comment by MarcosSpessatto** [comment]
@valmar can you please update your server version and try again? Thanks.

**Comment by valmar** [comment]
Dear Marcos,

   I tried and that didn't work. However, it worked this way:

- I did a new installation of Rocketchat
- Restored from the MongoDB dump of the previous installation *without
clearing the new mongdodb database*. This means that some tables are not
imported from the backup, because they already exist
- Restarted the Rocketchat server

This makes me think that there was a database corruption. Now it seems to
be working!

Thank you for your help

Valerio
